x-pack/plugins/lens/public/visualizations/xy/xy_config_panel/layer_header.tsx

```diff
@@ -231,18 +231,16 @@ const DataLayerHeaderTrigger = function ({
   return (
     <ToolbarButton
       data-test-subj="lns_layer_settings"
-      title={currentVisType.fullLabel || currentVisType.label}
+      aria-label={currentVisType.fullLabel || currentVisType.label}
       onClick={onClick}
       fullWidth
       size="s"
-      textProps={{ style: { lineHeight: '100%' } }}
-    >
-      <>
-        <EuiIcon type={currentVisType.icon} />
+      iconType={currentVisType.icon}
+      label={
```

Comment:
There is a slight change in Lens layer panel look, was that intended?

<img width="442" alt="Screenshot 2023-11-14 at 10 28 44" src="https://github.com/elastic/kibana/assets/4283304/342ed706-9806-41b1-bb26-06a7285adcb1">

<img width="427" alt="Screenshot 2023-11-14 at 10 28 40" src="https://github.com/elastic/kibana/assets/4283304/34ebbf40-5ce7-4055-ba2d-e38183913cc4">

If not, here's how you can correct it:
```
 <ToolbarButton
      data-test-subj="lns_layer_settings"
      aria-label={currentVisType.fullLabel || currentVisType.label}
      onClick={onClick}
      fullWidth
      size="s"
      iconType="arrowDown"
      label={
        <div>
          <EuiIcon type={currentVisType.icon} />
          <EuiText size="s" className="lnsLayerPanelChartSwitch_title">
            {currentVisType.fullLabel || currentVisType.label}
          </EuiText>
        </div>
      }
    />
```

Reply:
It wasn't, thanks for spotting this. I opted for a different markup that achieves pretty much the same thing, with a extra bit of ensuring the icon  for the selected visualisation is center aligned with the text. 
<img width="392" alt="Screenshot 2023-11-14 at 11 14 00" src="https://github.com/elastic/kibana/assets/7893459/ed09ec1b-aa44-4b93-bd16-07b224bec3db">